DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/experimental/platform_designer/README.md

```diff
@@ -0,0 +1,315 @@
+2. Choose pin `PIN_AM10` to drive the `i_clk` signal
+3. Choose pin `PIN_AR23` to drive the `fpga_led` signal
+4. Choose pin `PIN_AV21` to drive the `reset_button_n` signal
+5. Use `jtag.sdc` from the Intel® Arria® 10 SoC Golden Hardware Reference Design (GHRD) [source code](https://github.com/altera-opensource/ghrd-socfpga).
+
+## Building the `platform_designer` Tutorial
+
+> **Note**: When working with the command-line interface (CLI), you should configure the oneAPI toolkits using environment variables. 
+> Set up your CLI environment by sourcing the `setvars` script located in the root of your oneAPI installation every time you open a new terminal window. 
+> This practice ensures that your compiler, libraries, and tools are ready for development.
+>
+> Linux*:
+> - For system wide installations: `. /opt/intel/oneapi/setvars.sh`
+> - For private installations: ` . ~/intel/oneapi/setvars.sh`
+> - For non-POSIX shells, like csh, use the following command: `bash -c 'source <install-dir>/setvars.sh ; exec csh'`
+>
+> Windows*:
+> - `C:\Program Files(x86)\Intel\oneAPI\setvars.bat`
+> - Windows PowerShell*, use the following command: `cmd.exe "/K" '"C:\Program Files (x86)\Intel\oneAPI\setvars.bat" && powershell'`
+>
+> For more information on configuring environment variables, see [Use the setvars Script with Linux* or macOS*](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-linux-or-macos.html) or [Use the setvars Script with Windows*](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-windows.html).
+
+Follow these steps to compile and test the design:
+1. Compile the SYCL code to RTL using a Windows or Linux machine. 
+
+   Linux:
+
+   ```bash
+   $> cd add-oneapi
+   $> mkdir build
+   $> cd build
+   $> cmake ..
+   $> make fpga_ip_export
+   ```
+
+   Windows:
+
+   ```bash
+   > cd add-oneapi
+   > mkdir build
+   > cd build
+   > cmake -G "NMake Makefiles" ..
+   > nmake fpga_ip_export
+   ```
+
+   For more details, see the README in the `add-oneapi` directory.
+
+2. **From the same terminal**, launch the Intel® Quartus® Prime Pro Edition GUI, and create a new Intel® Quartus® Prime project using the 'New Project' wizard. 
+
+   Linux:
+   
+   ```
+   $> cd ../../add-quartus
+   $> quartus
+   ```
+
+   Windows:
+   
+   ```
+   > cd ..\..\add-quartus
+   > quartus.exe
+   ```
+
+   1. Set the project directory to be the `add-quartus` directory of this code sample.
+
+   2. Set the top-level entity to be `add` to make project management easier.
+
+      ![](README.md.assets/quartus_new_project.png)
+
+   4. Choose **Empty Project** when prompted to select a project type.
+
+   5. Add the source file `add.sv` and `jtag.sdc` to the design when the wizard prompts you.
+
+      ![](README.md.assets/add-files.png)
```

Comment:
Please move all the assets into an `asset` folder rather than `README.md.assets` to be consistent with the other samples.

Reply:
done